website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,103 @@
+$(document).ready(function () {
+	console.log(openerp);
+	$('.type_selection').each(function () {
+	    var type_selection = this;
```

Comment:
This probably does not do what you want. If you call `on()` on an element and pass a selector, it will look for *children* of that element matching the selector and react on events for those. You probably want to react on all events of that type, so this line should be 
```
$(type_selection).on('click', function()
```
And why 'click' and not 'change'?